README.md

```diff
@@ -19,7 +19,7 @@ Each package has its own **node_modules** directory that contains symbolic links
 ## Prerequisites
 
 - [Git](https://git-scm.com/)
-- [Node](https://nodejs.org/en/): an installation of the latest security patch of Node 16. The Node installation also includes the **npm** package manager.
+- [Node](https://nodejs.org/en/): an installation of the latest security patch of Node.js. Supported releases are 12.x, 14.x, 16.x, and 18.x. The Node installation also includes the **npm** package manager.
```

Comment:
I'd leave this to say the latest security patch of Node 18 since we'd want devs to use it normally which is different than our supported list.

Reply:
done